compiler/rustc_abi/src/lib.rs

```diff
@@ -1058,16 +1058,24 @@ pub enum Scalar {
 impl Scalar {
     #[inline]
     pub fn is_bool(&self) -> bool {
-        use Integer::*;
         matches!(
             self,
             Scalar::Initialized {
-                value: Primitive::Int(I8, false),
+                value: Primitive::Int(_, false),
```

Comment:
Should this still be called is_bool? Is a u64 that can only be 0 or 1 a bool? Normally I would say no.

Maybe call this `is_i1`? `is_bit`?

Reply:
`is_i1` probably is better yeah